Octokit/Clients/RepositoriesClient.cs

```diff
@@ -86,6 +86,11 @@ public Task<Repository> Create(string organizationLogin, NewRepository newReposi
                     errorMessage,
                     StringComparison.OrdinalIgnoreCase))
                 {
+                    if (string.IsNullOrEmpty(organizationLogin))
+                    {
+                        throw new RepositoryExistsException(newRepository.Name, e);
+                    }
```

Comment:
> This is due to fact that we don't have access to the owner's name if it's an account and not an organization.

Ah. And we don't have access to the name because when we do operations for the _current_ account, the name isn't needed as part of the URL. Right?

That was confusing to me and I should know that. :stuck_out_tongue: